packages/atomic-react/README.md

```diff
@@ -295,90 +296,94 @@ import {AtomicSearchInterface} from '@coveo/atomic-react';
+In the above example, you can see that of the time of this writing:
 
-To summarize, as of the time of this writing, the versions that we should keep in mind are the following:
+- The current latest version of `@coveo/atomic-react` is `2.1.35`.
+- The current latest version is using `@coveo/atomic` at version `2.19.16`.
+- The current latest version is using `@coveo/headless` at version `2.8.10`.
+- The current latest version is using `react`/`react-dom` at version 18 and above.
 
-- `@coveo/atomic-react` -> 1.23.11
-- `@coveo/atomic` -> 1.108.2
-- `@coveo/headless` -> 1.103.3
+Then, we need to add the required dependencies in our page. You can do that like so:
 
 ```html
 <head>
-  <!-- React and ReactDOM need to be included  -->
-  <script crossorigin src="https://unpkg.com/react@18/umd/react.production.min.js"></script>
-  <script crossorigin src="https://unpkg.com/react-dom@18/umd/react-dom.production.min.js"></script>
+  <!-- React, ReactDOM and ReactDOMServer need to be included  -->
+  <script
+    crossorigin
+    src="https://unpkg.com/react@18/umd/react.production.min.js"
+  ></script>
+  <script
+    crossorigin
+    src="https://unpkg.com/react-dom@18/umd/react-dom.production.min.js"
+  ></script>
+  <script
+    crossorigin
+    src="https://unpkg.com/react-dom@18/umd/react-dom-server-legacy.browser.production.min.js"
+  ></script>
 
   <!-- Optional script, which allows to use JSX directly in an inline script in the page -->
-  <script crossorigin src="https://unpkg.com/babel-standalone@6/babel.min.js"></script>
+  <script
+    crossorigin
+    src="https://unpkg.com/@babel/standalone@7/babel.min.js"
+  ></script>
 
   <!-- @coveo/headless need to be included as a dependency -->
   <!-- Note the matching major and minor version as explained above -->
-  <script crossorigin src="https://static.cloud.coveo.com/headless/v1.103/headless.js"></script>
+  <script
+    crossorigin
+    src="https://static.cloud.coveo.com/headless/v2.8/headless.js"
+  ></script>
 
-  
   <!-- @coveo/atomic need to be included as a dependency -->
   <!-- Note the matching major and minor version as explained above -->
-  <script crossorigin type="module" src="https://static.cloud.coveo.com/atomic/v1.108/atomic.esm.js"></script>
-  
+  <script
+    crossorigin
+    type="module"
+    src="https://static.cloud.coveo.com/atomic/v2.19/atomic.esm.js"
+  ></script>
+
   <!--And then finally @coveo/atomic-react is included -->
   <!-- Note the matching major and minor version as explained above -->
-  <script crossorigin src="https://static.cloud.coveo.com/atomic-react/v1.23/iife/atomic-react.min.js"></script>
+  <script
+    crossorigin
+    src="https://static.cloud.coveo.com/atomic-react/v2.1/iife/atomic-react.min.js"
+  ></script>
 </head>
 ```
 
-Once this is done, you can start using `CoveoAtomicReact` directly with an inline script tag: 
+Once this is done, you can start using `CoveoAtomicReact` directly with an inline script tag:
 
 ```html
```

Comment:
I made some changes to shorten this sample a little bit, I can revert them if you prefer.